std/os/linux/test.zig

```diff
@@ -42,3 +44,39 @@ test "timer" {
     // TODO implicit cast from *[N]T to [*]T
     err = linux.epoll_wait(@intCast(i32, epoll_fd), @ptrCast([*]linux.epoll_event, &events), 8, -1);
 }
+
+export fn iter_fn(info: *linux.dl_phdr_info, size: usize, data: ?*usize) i32 {
+    var counter = data orelse @panic("???");
```

Comment:
Could just be `orelse unreachable` I think?

Reply:
Good catch, I went with `.?`